pymc_marketing/mmm/validating.py

```diff
@@ -40,6 +48,12 @@ class ValidateDateColumn:
     def validate_date_col(self, data: pd.DataFrame) -> None:
         if self.date_column not in data.columns:
             raise ValueError(f"date_col {self.date_column} not in data")
+
+        if not _has_valid_indices(data):
+            raise ValueError(
+                "X or y has incorrect indices. Try to reset with `data.reset_index(inplace=True)`"
+            )
```

Comment:
nitpick: this is small enough that i would just include the code here instead of in the hidden function.

Reply:
Fixed